src/asana/helpers.py

```diff
@@ -309,6 +309,31 @@ def urlreplace(matchobj: Match[str]) -> str:
     return re.sub(URL_REGEX, urlreplace, text)
 
 
+def get_linked_task_ids(pull_request: PullRequest) -> List[str]:
+    """
+    Extracts linked task ids from the body of the PR.
+    """
+    body_lines = pull_request.body().splitlines()
+    stripped_body_lines = (line.strip() for line in body_lines)
+    task_url_line = None
+    seen_asana_tasks_line = False
+    for line in stripped_body_lines:
+        if seen_asana_tasks_line:
+            task_url_line = line
+            break
+        if line.startswith("Asana tasks:"):
+            seen_asana_tasks_line = True
+
+    if task_url_line:
+        task_urls = task_url_line.split()
+        task_ids = []
+        for url in task_urls:
+            task_ids.append(re.findall("\d+(?!.*\d)", url)[0])
```

Comment:
This is making an assumption that the task links will be in the format we expect. Can you make this code a bit safer by first running the regex, then if it matches, extract `[0]` from the result? Adding a test where the line is something like `"Asana tasks:\nfoo bar"` should be a good starting point from a TDD route